client/packages/requisitions/src/ResponseRequisition/DetailView/columns.ts

```diff
@@ -131,7 +148,6 @@ export const useResponseColumns = () => {
       getItemId: row => row.itemId,
       getQuantity: row => NumUtils.round(row.remainingQuantityToSupply),
     }),
-    width: 100,
```

Comment:
This was fixed in another issue (which was causing the select boxes to overlap with last column) but was added back in PV?